### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bdmoore1**
# Description
Added style guide to the README for those who contribute to documentation.

Changes proposed in this pull request:
- Styles section in README
-
-

### CODE REVIEW

**Comment by bdmoore1** [comment]
@PivovarA do you know why CI is failing?

**Comment by Alexsandruss** [comment]
I guess CI fails might be ignored for docs only PRs.
@outoftardis, can you verify that change is meaningful and suitable for README.md (not another file like CONTRIBUTING.md, for example)?

**Comment by outoftardis** [comment]
> I guess CI fails might be ignored for docs only PRs. @outoftardis, can you verify that change is meaningful and suitable for README.md (not another file like CONTRIBUTING.md, for example)?

You are right. CONTRIBUTING.md is a better place for this info

**Review by Alexsandruss — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+   ---------
+
+   Heading 2
+   *********
+
+   Heading 3
+   =========
+
+   Heading 4
+   ^^^^^^^^^
```
Comment (Alexsandruss) [inline CONTRIBUTING.md:24]: This block is automatically formatted in Markdown and will look confusing if one reads .md file not in raw mode, insert it as Markdown code block using:

\`\`\`
[RST formatting examples]
\`\`\`

**Review by Alexsandruss — CHANGES_REQUESTED** [review]

**Comment by napetrov** [comment]
Better version of contributing.md was intergrated

**Pull request closed without merging**